src/outagesim/backup_reliability.jl

```diff
@@ -505,76 +765,115 @@ function backup_reliability_inputs(d::Dict, p::REoptInputs; r::Dict = Dict())::D
     end
```

Comment:
batt_charge_efficiency and batt_discharge_efficiency created but not used? (A few lines above, I can't put a comment there because those changes aren't in this PR.)

Reply:
Need to be added to the dict r2 returned, otherwise arg default is always used later on

Reply:
Also the loop writes over the same variable for each b in p.s.storage.types.elec

Reply:
@sericson0 this doesn't appear to be addressed yet

Reply:
Added the battery charge and discharge efficiencies to r2. Not sure how best to address the problem of having multiple storage types. Would it make sense to only allow a single storage type?

Reply:
resolving because only allowing one elec storage type is good